test/e2e/drc_crud/drc/framework.go

```diff
@@ -26,40 +26,40 @@ type Framework struct {
 
 // NewFramework create a new Framework with name
 func NewFramework(name string) *Framework {
-	namespace := fmt.Sprintf("drce2e-%s-%s", name, RandString(8))
+	namespace := fmt.Sprintf("drce2e-%s-%s", name, utils.RandString(8))
 	return &Framework{
 		nameSpace: namespace,
 	}
 }
 
 // BeforeEach runs before each test
 func (f *Framework) BeforeEach() error {
-	// Get a config to talk to the apiserver
+	// Get a config to talk to the api-server
 	cfg, err := config.GetConfig()
 	if err != nil {
 		f.Failf("get k8s config err: %s", err)
 	}
 	f.KubeConfig = cfg
-	client, err := testclient.NewClient(cfg)
+	testClient, err := NewClient(cfg)
 	if err != nil {
 		f.Failf("get k8s config err: %s", err)
 	}
-	f.Client = client
+	f.Client = testClient
 	if err := f.createTestNamespace(); err != nil {
 		return err
 	}
-	if err := f.createRBAC(); err != nil {
-		return err
-	}
+	//if err := f.createRBAC(); err != nil {
```

Comment:
I'd leave these. AFAIK, there's no harm in having them. But I assume one part of the test might be dependent on them